lib/src/components/NxModal/types.ts

```diff
@@ -7,7 +7,7 @@
 import {HTMLAttributes} from 'react';
 import * as PropTypes from 'prop-types';
 
-export type CloseHandler = (evt: Event) => void;
+export type CloseHandler = (evt?: Event) => void;
```

Comment:
This _is_ a breaking change

Reply:
> This _is_ a breaking change

Fixed here: 7ba614eafb29907d249ec237bdb4f511c2c7f212